storybook/stories/ComponentLists.tsx

```diff
@@ -29,10 +29,17 @@ import {
   StakeData,
   Votes,
   Vote,
-  VoteData
+  VoteData,
+  Loader,
 } from "../../src/";
 import ComponentListView, { PropertyType } from "../helpers/ComponentListView";
 
+class Test extends React.Component {
```

Comment:
Pure testing, I left this so you can see how it's working, after testing please let me know and I will change it @dOrgJelli

Reply:
Let's make an actual test for this, and a section in the storybook to test this